File: CPP/Aksesoris.cpp
//jenis, bahan, warna

#include <iostream>
#include <string>
#include "PetShop.cpp"

using namespace std;

class Aksesoris : public PetShop{
    private:
        string jenis;
        string bahan;
        string warna;
    
    public:
        Aksesoris(){
            setID("");
            setNama("");
            setHarga("");
            setStok("");
            jenis = "";
            bahan = "";
            warna = "";
        };

        Aksesoris(string ID, string Nama, string Harga, string Stok, string Jenis, string Bahan, string Warna){
            setID(ID);
            setNama(Nama);
            setHarga(Harga);
            setStok(Stok);
            jenis = Jenis;
            bahan = Bahan;
            warna = Warna;
        };

        void setJenis(string Jenis){
            jenis = Jenis;
        };

        void setBahan(string Bahan){
            bahan = Bahan;
        };

        void setWarna(string Warna){
            warna = Warna;
        };

        string getJenis(){
            return jenis;
        };

        string getBahan(){
            return bahan;
        };

        string getWarna(){
            return warna;
        };

        ~Aksesoris(){
        }
};

File: PHP/Aksesoris.php
<?php
include 'PetShop.php';
class Aksesoris extends PetShop{
    private $jenis = "";
    private $bahan = "";
    private $warna = "";
    function __construct($id, $nama, $harga, $stok, $foto, $jenis, $bahan, $warna){
        $this->setID($id);
        $this->setNamaProduk($nama);
        $this->setHargaProduk($harga);
        $this->setStokProduk($stok);
        $this->setFotoProduk($foto);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }
    function setJenis($jenis){
        $this->jenis = $jenis;
    }
    function setBahan($bahan){
        $this->bahan = $bahan;
    }
    function setWarna($warna){
        $this->warna = $warna;
    }
    function getJenis(){
        return $this->jenis;
    }
    function getBahan(){
        return $this->bahan;
    }
    function getWarna(){
        return $this->warna;
    }
}
?>